src/app/space-wizard/common/logger.ts

```diff
@@ -8,7 +8,7 @@ export interface ILogEntry {
   inner?: any;
 }
 export interface ILoggerDelegate {
-  (options: string | ILogEntry): void;
+  (options: string | ILogEntry,...args): void;
```

Comment:
Why did you use ...args? This is an interface, that not accessed and it is an optional field(s).